### ISSUE TITLE
Request: Umbrello the UML diagram program

### ISSUE THREAD

**Issue by wicast** [KDE-mac/homebrew-kde#296]
Umberello seems to be what I want to use for uml drawing. Do you guys consider it?

**Comment by yurikoles** [comment]
We'll definitely consider a PR ;)

**Comment by sl1pkn07** [comment]
https://github.com/KDE-mac/homebrew-kde/tree/umbrello

**Comment by wicast** [comment]
I can't build kdevelop successfully,lol.
```
CMake Error at /usr/local/Cellar/cmake/3.14.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find SharedMimeInfo (missing: UPDATE_MIME_DATABASE_EXECUTABLE)
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.14.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/share/ECM/find-modules/FindSharedMimeInfo.cmake:74 (find_package_handle_standard_args)
  CMakeLists.txt:84 (find_package)
```

**Comment by sl1pkn07** [comment]
ow men....

**Comment by wicast** [comment]
kf5-kdelibs4support is one of the necessary dependency, I'm still trying

**Comment by wicast** [comment]
now is llvm, orz
```
../unittests/testllvmparser.cpp:49:75: error: no member named 'getLocStart' in 'clang::CXXRecordDecl'
            FullSourceLoc FullLocation = Context->getFullLoc(Declaration->getLocStart());
                                                             ~~~~~~~~~~~  ^
1283 warnings and 1 error generated.
ninja: build stopped: subcommand failed.

```

**Comment by wicast** [comment]
By adding command `system "sed", "-i", "--" ,"/BUILD_UNITTESTS 1/d", "../CMakeLists.txt"`, I have made it.
But the experience of ui is .... quite terrible I have to say. It is not fitting well for the osx.
by the way it is
`      ln -sfv "$(brew --prefix)/opt/umbrello/bin/umbrello5.app" "$HOME/Applications/KDE/"
`

**Comment by wicast** [comment]
![image](https://user-images.githubusercontent.com/2603937/57287249-b3ef3500-70e9-11e9-8702-513bc5a69c2c.png)
It's not looking good in osx, orz.

**Comment by sl1pkn07** [comment]
> now is llvm, orz
> 
> ```
> ../unittests/testllvmparser.cpp:49:75: error: no member named 'getLocStart' in 'clang::CXXRecordDecl'
>             FullSourceLoc FullLocation = Context->getFullLoc(Declaration->getLocStart());
>                                                              ~~~~~~~~~~~  ^
> 1283 warnings and 1 error generated.
> ninja: build stopped: subcommand failed.
> ```



> By adding command `system "sed", "-i", "--" ,"/BUILD_UNITTESTS 1/d", "../CMakeLists.txt"`, I have made it.
> But the experience of ui is .... quite terrible I have to say. It is not fitting well for the osx.
> by the way it is
> ` ln -sfv "$(brew --prefix)/opt/umbrello/bin/umbrello5.app" "$HOME/Applications/KDE/"`

i haven't that error. build without problems, also, the unittest it was disabled disabled in the args

greetings

**Comment by sl1pkn07** [comment]
> kf5-kdelibs4support is one of the necessary dependency, I'm still trying

fixed

**Comment by yurikoles** [comment]
fixed in 9c3927b8eae59ef077504ffea9a3a88a2e9730cc

**Comment by yurikoles** [comment]
@sl1pkn07 please refer to FR tickets in PRs :)
